platform/broadcom/saibcm-modules/systems/linux/kernel/modules/psample/psample.c

```diff
@@ -0,0 +1,302 @@
+/*
+ * net/psample/psample.c - Netlink channel for packet sampling
```

Comment:
there are two psample.c, which one are we using? can you remove the one that is not using?

Reply:
There is a psample.c file in the Linux kernel source but I am not finding it getting build when building a sonic Broadcom image. Is it getting build only for VS image? I found certain psample module handling for vs the file "files/build_templates/sonic_debian_extension.j2"

Reply:
Same comment as below, closing one of them

Reply:
it is build and you can find it here.

admin@vlab-01:/lib/modules/4.9.0-9-2-amd64$ sudo find . -name "psample*"
./kernel/net/psample
./kernel/net/psample/psample.ko

Reply:
This is same as the below comment discussion.